grails-app/controllers/org/pih/warehouse/api/InventoryApiController.groovy

```diff
@@ -59,4 +65,24 @@ class InventoryApiController {
             }
         }
     }
+
+    def getExpirationHistoryReport(ExpirationHistoryReportFilterCommand command) {
+        if (command.hasErrors()) {
+            throw new ValidationException("Invalid filters", command.errors)
+        }
+        withFormat {
+            "csv" {
+                String csv = inventoryService.getExpirationHistoryReportCsv(command)
+                response.contentType = "text/csv"
+                String filename = "Expiration history report - ${AuthService.currentLocation?.name}.csv"
```

Comment:
If you're including `facilityId` in the URI you should probably use that instead

Reply:
It's actually a good point if we should. I mean, if we should use the facility in the URI... I did it as it was pointed out during the last PR, but there is a requirement in the ticket that the location is always supposed to be the current location, so I'm wondering if we should force to have the facility id in the URI (and then have to validate that if the one provided in the URI is your current one), or just get rid of that in the URI and keep using currentLocation in the code like it is now.